lib/Core/Executor.cpp

```diff
@@ -2904,7 +2903,7 @@ void Executor::callExternalFunction(ExecutionState &state,
       if (i != arguments.size()-1)
 	os << ", ";
     }
-    os << ")";
+    os << ") at " << target->info->file << ":" << target->info->line;
```

Comment:
@ccadar Feel free to remove the spaces. Beside that, maybe we can move `Executor::printFileLine()` to `KInstruction` as the `state` argument is not used anyway.